src/lib/viewers/controls/controls-root/ControlsRoot.scss

```diff
@@ -4,6 +4,7 @@
     position: absolute;
     bottom: 25px;
     left: 50%;
+    z-index: 3;
```

Comment:
Why is this needed?

Reply:
because the text layer z-index was set to 2 in this release and it was covering the buttons

Reply:
because the text layer z-index was set to 2 in this release and it was covering the buttons

Reply:
because the text layer z-index was set to 2 in this release and it was covering the buttons